src/cmd/flux-job.c

```diff
@@ -3441,28 +3511,45 @@ void info_lookup (flux_t *h,
     while (optindex < argc) {
         json_t *s;
         const char *key = argv[optindex];
+        const char *extra_key = NULL;
 
         if (!(s = json_string (key)))
             log_msg_exit ("json_string");
         if (json_array_append_new (ctx.keys_input, s) < 0)
             log_msg_exit ("json_array_append");
 
-        /*  Special case: if --original was used and the key is "jobspec",
-         *   then fetch J and decode it on behalf of the caller.
-         */
-        if (optparse_hasopt (p, "original") && streq (key, "jobspec")) {
-            ctx.original = true;
-            key = "J";
+        /*  Special cases for key "jobspec" */
+        if (streq (key, "jobspec")) {
+            /* if --original was used fetch J and decode it on behalf of the caller. */
+            if (optparse_hasopt (p, "original")) {
+                ctx.original = true;
+                key = "J";
+            }
+            else if (optparse_hasopt (p, "base"))
+                ctx.base = true;
+            else
+                /* also get eventlog to build viewed jobspec */
+                extra_key = "eventlog";
         }
 
         /* N.B. job-info.lookup will ignore duplicate keys, in the
-         * event user specified both "jobspec" and "J" on the command
-         * line with --original.
+         * event user specified duplicate keys or options that lead to
+         * duplicate keys: e.g.
+         * - --original and both "jobspec" and "J"
+         * - --base and both "jobspec" and "eventlog"
```

Comment:
From the code it appears that "jobspec" and "eventlog" are both fetched only if _not_ `--base` not with `--base`.
Is this comment correct?